action.yaml

```
@@ -23,7 +23,9 @@ runs:
 
         echo ::group::Install dependencies
         # Remove this package from constraints
-        PKG_NAME=$(pip list --local --editable --format json | jq '.[0].name' -r)
+        PKG_NAME=$(
+          pip list -l -e --format json | jq '.[0].name' -r | sed 's/_/-/g'
```

```suggestion
          pip list -l -e --format json | jq '.[0].name' -r | tr _ -
```

Using `tr` for a straighthforward character replacement might be easier on re-reads but these are literally equivalent so this is a very soft suggestion.

---

Nice!